utils/rng-tools/Makefile

```diff
@@ -34,24 +36,19 @@ define Package/rng-tools
 endef
 
 define Package/rng-tools/description
-Daemon for adding entropy to kernel entropy pool. By default it uses
-/dev/urandom as the source but the init script can be modified
-to use a hardware source like /dev/hwrng if present
+  Daemon for adding entropy to kernel entropy pool. By default it uses
+  /dev/urandom as the source but the init script can be modified
+  to use a hardware source like /dev/hwrng if present
 endef
 
-ifdef CONFIG_USE_UCLIBC
-CONFIGURE_VARS += \
-    LIBS="-largp"
-endif
-
-ifdef CONFIG_USE_MUSL
-CONFIGURE_VARS += \
-    LIBS="-largp"
-endif
-
 CONFIGURE_ARGS += \
 	--without-libgcrypt \
-	--without-nistbeacon
+	--without-nistbeacon \
+	--without-pkcs11
+
+ifndef CONFIG_USE_GLIBC
+	CONFIGURE_VARS += LIBS="-largp"
+endif
```

Comment:
Maybe preserve the original `ifdef CONFIG_USE_UCLIBC` and `ifdef CONFIG_USE_MUSL` tests (and similarly above with the `PKG_BUILD_DEPENDS`) rather than assuming that any non-glibc libc will need `-largp`?  I'm not adamant, but if we have three now, we might one day have four...

Other than that one nit, this looks fine to me.

Reply:
I would think the number to drop to 2. uClibc-ng is kept around only for ARC support if I'm not mistaken.

I'll see if I can come up with a cleaner solution.

Reply:
OK.  Don't spend too much time on it.  If the rest of the tree is ndef glibc for these things, then follow along.

Reply:
From a quick Google search, it seems argp is only part of glibc. I couldn't find it being part of any other libc. Seems it's a GNU extension.